Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -315,43 +315,16 @@ final class PurchasesOrchestrator {
 
     func purchase(product: StoreProduct,
                   package: Package?,
-                  completion: @escaping PurchaseCompletedBlock) {
```

Comment:
took the opportunity to simplify a bit the purchase methods in `PurchaseOrchestrator` and removed the method to purchase a package without `promotionalOffer`, instead opting to make `promotionalOffer` and `metadata` optional in the method below and handling the routing to SK1 and SK2 appropriately.

Reply:
~~Hmm is this a problem for ObjC? Just making sure we don't add a breaking change there accidentally. But I think it makes sense!~~

Edit: Nvm, it doesn't break the API tests and I think it should be fine 👍

Reply:
Yeah this method is private from PurchasesOrchestrator, so it's just an internal refactor.